tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
```

Comment:
Hello, please set the  "allowJs" attribute here to false

Reply:
Okay understood